samples/star-wars-api/Policies.fs

```diff
@@ -0,0 +1,26 @@
+namespace FSharp.Data.GraphQL.Samples.StarWarsApi.Authorization
+
+open FSharp.Core
+open Microsoft.AspNetCore.Authorization
+
+module Policies =
+
+    let [<Literal>] CanSetMoon = "CanSetMoon"
+
+type IsCharacterRequierment (character : string list) =
+    member val Characters = character
```

Comment:
Because this Characters is used for searching (List.contains below) then it could be beneficial to have the property as Set instead of List (e.g. just `|> Set.ofList` here). To instantiate a Set is a bit slower than a list, but then when you do `.contains` (like you do in a loop below) then Set is faster for searching.

Reply:
Fixed, thanks!